templates/todo/projects/python-mongo/README.md

```diff
@@ -1,8 +1,9 @@
-# PLACEHOLDER_TITLE
+# $PLACEHOLDER_TITLE
```

Comment:
@wbreza What do you think about splitting the `terraform` vs `bicep` README files out? Just wanted to see if you agree.

I am worried that the number of substitutions will continue increasing. I have #1556 that will be doing this, and since I need to address the merge conflicts anyway, it wouldn't be the hassle.

Reply:
I feel like if we split out the README's there will be more maintenance when we need to make any copy edits or other changes.  Right now we do kind of a hacky repoman replacement strategy but would be interesting to build out some direct support in repoman for intra-file replacements.  

Not going to address this for this PR.